lib/fluent/load.rb

```diff
@@ -15,6 +15,8 @@
   # ignore setup error on Win or similar platform which doesn't support signal
 end
 require 'cool.io'
+
+$platformwin = /mswin|mingw/ === RUBY_PLATFORM
```

Comment:
From sada's comment on serverengine, how about using this approach like `Fluent.on_windows?` ? > https://github.com/fluent/serverengine/pull/26#discussion_r40830182

We have fluent/env.rb so this file seems good place.